tests/backend/workflows/test_mam_status_workflow.py

```
@@ -0,0 +1,62 @@
+        "label": "seedbox",
+        "mam": {
+            "mam_id": "mam-cookie",
+            "session_type": "ip",
+            "ip_monitoring_mode": "static",
+        },
+        "mam_ip": "198.51.100.10",
+        "proxy": {},
+    }
+    assert (await api_client.post("/api/session/save", json=session)).is_success
+
+    async def ipinfo(*_args: Any, **_kwargs: Any) -> dict[str, str]:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*_args`

(ANN401)